exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,317 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In this case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
```

Comment:
We don't have a concurrency test among the tests, mostly because I am a little bit unsure how you would write one since most of the time that includes tricking with loops and I don't know how different language deals with that. But perhaps we should.

Reply:
We should definitely have a concurrency test I feel! It's what this exercise is mostly about. How a language implements it is not of any real concern to this canonical data, so we don't really need to consider that. 

BTW: this is how C# implements it https://github.com/exercism/csharp/blob/main/exercises/practice/bank-account/BankAccountTests.cs#L61-L84

Reply:
Yeah. I left it out since i was unsure how to add it, and also there are languages that doesn't really have concurrency. But if you have a suggestion on how a concurrency tests data should look like so am I up for suggestions.

Reply:
@meatball133 What do you think about the above suggestion? 

p.s. We have a `concurrent` scenario now

Reply:
I think that could work.